library/src/test/resources/application-test.yml

```diff
@@ -52,9 +52,7 @@ landingzone:
       - Syslog
     longTermStorageAccountIds:
       #extend configuration in case mrg is in different region; resource id should be fully qualified
-      eastus2: /subscriptions/c5f8eca3-f512-48cb-b01f-f19f1af9014c/resourceGroups/terra-dev-protected-data-logs-rg/providers/Microsoft.Storage/storageAccounts/ltslogsdeveastus2
-      eastus: /subscriptions/c5f8eca3-f512-48cb-b01f-f19f1af9014c/resourceGroups/terra-dev-protected-data-logs-rg/providers/Microsoft.Storage/storageAccounts/ltslogsdeveastus
-      southcentralus: /subscriptions/c5f8eca3-f512-48cb-b01f-f19f1af9014c/resourceGroups/terra-dev-protected-data-logs-rg/providers/Microsoft.Storage/storageAccounts/ltslogsdevsthcntrlus
+      southcentralus: /subscriptions/df547342-9cfd-44ef-a6dd-df0ede32f1e3/resourceGroups/landing-zone-testing/providers/Microsoft.Storage/storageAccounts/ltssthcentralus
```

Comment:
This is pointing CI at a resource group in the workspaces-dev subscription. Terraforming up the needed service principal auth was proving quite difficult (nested loops over environments + regions), this seems like a good compromise.